### DISCUSSION TITLE
"autoCatalog" doesn't make contents.

### DISCUSSION THREAD

**Discussion by Someijam** [vuepress-theme-hope/vuepress-theme-hope#3602] [Q&A] [answered]
My theme.ts(was imported by config.ts) is as follows:
```
import { hopeTheme } from "vuepress-theme-hope";
import navbar from "./navbar.js";
import sidebar from "./sidebar.js";

export default hopeTheme({
 ......

plugins: {
    ...
    autoCatalog:{
      //options...
    },
    ... 
},
.......
```

However, when I run "yarn docs:dev", there's nothing like contents in every entry and every "README.md" in every entry does't contain something like this.
1. If I don't put "README.md", it won't generate one when I run "yarn docs:dev.
2. If there's already a "README.md" I created before, nothing changed when I run "yarn docs:dev.

Did I do anything wrong in "theme.ts" or I have to do something with my "README.md"?

Current README.md
```
---
title: My Title
icon: lightbulb
---

# Welcome to "My Title"
```

**Comment by Mister-Hope** [comment]
I need a minimal reproduction, please give me one

**Reply by Someijam** [reply]
Just the Default template with some modifications:
1. Change sidebar to "/":"structure".
2. Modified "README.md" in docs/demo/ and docs/guide/.
3. enabled "autoCatalog" in theme.ts.

For example :`http://localhost:8080/demo/` doesn't contain a catalog.
(I think I may do something wrong, so just tell me how to generate "README.md"s with catalog in every directory will also be OK. )
Here, I attach a sample without "node_modules".
[demovue.zip](https://github.com/vuepress-theme-hope/vuepress-theme-hope/files/13261728/demovue.zip)

**Reply by Mister-Hope** [reply]
I'm busy now,  I will have a look at your question later in the middle.of.the week

**Reply by Mister-Hope** [reply, marked as answer]
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/5fc51dc8-7e4d-4e11-b005-67a06fd0866c)

I delete `guide/README.md` in docs template and I can get a catalog page?


The auto catalog only works if the README.md file not exists, if it is, you shall manually add `<AutoCatalog />` component to markdown file.
